.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,37 @@
+name: Feature Request 🚀
```

Comment:
Overall, I don't think having separate fields here help a lot. Something like description+example would be good enough for me.